pkg/container/kubernetes_runner.go

```diff
@@ -396,12 +396,16 @@ func (k *k8s) filterMounts(mount BindMount) bool {
 		return true
 	}
 
+	if mount.Destination == "/var/cache/melange" {
```

Comment:
Is there a constant that we can reference here?

Reply:
there weren't, but I added some!